assets/elm/main.elm

```diff
@@ -15,9 +13,14 @@ import Platform.Cmd
 -- MAIN
 
 
-main : Program Never Model Msg
+type alias Flags =
+    { socketServer : String
+    }
+
+
+main : Program Flags Model Msg
 main =
-    Html.program
+    Html.programWithFlags
```

Comment:
this is a way to pass configs (from the wrapper JS) into the Elm program at startup